packages/dd-trace/src/scope.js

```diff
@@ -58,6 +58,10 @@ class Scope {
   _isPromise (promise) {
     return promise && typeof promise.then === 'function'
   }
+
+  isNoop () {
```

Comment:
A `noop` scope is an internal concept, but this exposes it publicly. If anything this should be moved out of scope completely and moved to its own store.